File: ecos-records-spring/src/main/java/ru/citeck/ecos/records2/spring/config/RecordsAutoConfiguration.java
package ru.citeck.ecos.records2.spring.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

/**
 * Auto configuration to initialize records beans.
 *
 * @author Roman Makarskiy
 */
@Configuration
@ComponentScan(basePackages = {"ru.citeck.ecos.records2.spring"})
public class RecordsAutoConfiguration {
}

File: ecos-records-spring/src/main/java/ru/citeck/ecos/records3/spring/config/RecordsAutoConfiguration.kt
package ru.citeck.ecos.records3.spring.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

/**
 * Auto configuration to initialize records beans.
 *
 * @author Roman Makarskiy
 */
@Configuration
@ComponentScan(basePackages = {"ru.citeck.ecos.records3.spring"})
public class RecordsAutoConfiguration {
}
